bin/weewx/drivers/vantage.py

```diff
@@ -1110,14 +1212,27 @@ def hardware_name(self):
     def archive_interval(self):
         return self.archive_interval_
     
+    def determine_hardware(self):
```

Comment:
May be this change is not needed anymore. At first I got very regular a time out returned here (o bytes received) but after I implemented this change I realised there was no wakeup_console call present between the startup of the driver and this first command. I think we should add extra parameters to the read routines that default to the original behaviour (no time-out results or zero bytes as return). Only DMPAFT and LOOP will use this parameter to get a time out result back (in one way or another).